templates/web/merton/waste/enquiry-problem.html

```diff
@@ -0,0 +1,16 @@
+[%
+
+f = [];
+service_id = c.req.params.service_id;
+service = services.$service_id;
```

Comment:
These values aren't used; think these come from Bromley version, but e.g. Borsetshire's is a bit simpler, you don't need to construct an `f`, you could only have:
```
[% PROCESS waste/enquiry.html
    title = 'Report a problem'
    field_options = [
        { value = 'Bin not returned', label = 'Bin not returned correctly' },
        { value = 'Waste spillage', label = 'Waste spillage' },
    ]
%]
```